app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "from@example.com"
```

Comment:
actually, use this helper method:
```ruby
def mailer_suffix
  Rails.env.production? ? "" : "_#{Rails.env}"
end```
and set the default from to `"snowman#{mailer_suffix}@wediko.org"`